### ISSUE TITLE
RP DA scheduler fails to create VE on Comet

### ISSUE THREAD

**Issue by vivek-bala** [radical-cybertools/radical.pilot#1683]
Output in bootstrapper:
```
# -------------------------------------------------------------------
# Touching output tarballs
# -------------------------------------------------------------------
create gtod
build gtod with cc... success
0.0019,bootstrap_0_start,bootstrap_0,MainThread,pilot.0000,PMGR_ACTIVE_PENDING,
VIRTENV : /home/vivek91/radical.pilot.sandbox/ve.xsede.comet.0.50.1
VIRTENV : /home/vivek91/radical.pilot.sandbox/ve.xsede.comet.0.50.1 (normalized)
PYTHON: /opt/python/bin/python
PIP   : /opt/python/bin/pip
0.0606,ve_setup_start,bootstrap_0,MainThread,pilot.0000,PMGR_ACTIVE_PENDING,
virtenv_create   : TRUE
virtenv_update   : FALSE
rp install sources:  radical.utils-0.50.1-v0.50.1-1-gce60836-devel/ saga-python-0.47.6-v0.47.6-35-gf578245-devel/ radical.pilot-0.50.1-v0.50.1-92-g2542855-project-DA-scheduler-algo/
rp install target : SANDBOX
rp install lock   : FALSE
rp lock for ve create
obtained lock /home/vivek91/radical.pilot.sandbox/ve.xsede.comet.0.50.1.lock
2.0382,ve_create_start,bootstrap_0,MainThread,pilot.0000,PMGR_ACTIVE_PENDING,

# -------------------------------------------------------------------
#
# Download virtualenv tgz
# cmd: curl -k -L -O 'https://pypi.python.org/packages/source/v/virtualenv/virtualenv-1.9.tar.gz'
#

curl: (35) SSL connect error
#
# ERROR
# no fallback command available
#
# -------------------------------------------------------------------
WARNING: Couldn't download virtualenv via curl! Using system version.

# -------------------------------------------------------------------
#
# Create virtualenv
# cmd: virtualenv /home/vivek91/radical.pilot.sandbox/ve.xsede.comet.0.50.1
#
/home/vivek91/radical.pilot.sandbox/rp.session.vivek-HP-Pavilion-m6-Notebook-PC.vivek.017742.0042/bootstrap_0.sh: line 512: virtualenv: command not found
#
# ERROR
# no fallback command available
#
# -------------------------------------------------------------------
ERROR: Couldn't create virtualenv
Error on virtenv creation -- abort
removed `/home/vivek91/radical.pilot.sandbox/ve.xsede.comet.0.50.1.lock'

```

**Comment by vivek-bala** [comment]
Note that there is no VE in the sandbox prior to this execution. Although the default is to create a new VE for comet.
```
virtenv_create   : TRUE
```

**Comment by SrinivasMushnoori** [comment]
I face the same issue on Comet.

**Comment by andre-merzky** [comment]
Bugger, I just deleted the fix proposed for #1652, which looks very much like the same issue.  Vivek, can you please check if adding `-1` to the curl command in the bootstrapper changes anything?

**Comment by andre-merzky** [comment]
*sigh* allright then, will fix...  Thanks!!

**Comment by andre-merzky** [comment]
This is now fixed in master and devel, and released as `0.50.6`
